### PR TITLE
Jira DOC-746: RS - Document INE and new rladmin settings.

### PR DESCRIPTION
**Pull request by lanceleonard**
- Initial draft
- Also documents new MTLS flag (mtls_allow_no_xkusage)

### CODE REVIEW

**Comment by lanceleonard** [comment]
A preview of this work [is available](http://localhost:1313/rs/references/rladmin/#tune).

File: content/rs/security/internode-encryption.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+## Certificate authority and rotation
+
+Starting with v6.2.4, internode communication is managed, in part, by two certificates, one for the control plane and one for the data plane.  These certificates are signed by a private certificate authority (CA).  The CA is not exposed outside of the cluster, so it cannot be accessed by external processes or services.  In addition, each cluster generates a unique CA that is not used anywhere else.
+
+The private CA is generated when a cluster is created or upgraded to 6.2.4.  
+
+When nodes join the cluster, the cluster CA is used to generate certificates for the new node, one for each plane.  Certificates signed by the private CA are not shared between clusters and they're not exposed outside the cluster.
+
+All certificates signed by the internal CA are automatically rotated. Periodically, an internal process checks certificate expiration and generates new certificates when the existing ones are about to expire.  Alerts also monitor certificate expiration and are triggered when problems prevent certificates from being rotated.
```
Comment (AlonMagrafta) [inline content/rs/security/internode-encryption.md:61]: Do we want to state numbers? Rotation each 30 days

Comment (AlonMagrafta) [reply content/rs/security/internode-encryption.md:61]: And do we also want to add the API to manually initiate rotation? I think that given that we explained all the above we should.
put /v1/cluster/certificates/rotate

File: content/rs/references/rladmin.md [hunk 2]
```diff
@@ -702,6 +708,8 @@ rladmin tune cluster
 | default_concurrent_restore_actions | Default number of concurrent actions during node restore from a snapshot (positive integer or "all") |
 | show_internals |  |
 | expose_hostnames_for_all_suffixes |  |
+| redis_upgrade_policy | The policy defines how Redis upgrades are handled when you upgrade Redis Enterprise Software.<br /><br />Supported values are `latest` (Limits Redis upgrades to the current minor Redis release (`.x`) \[_default as of v6.2.4_] and `major` (Limits Redis upgrades to the current major (`.0`) release). \[_effective default prior to v6.2.4_] |
+| default_redis_version <value> | The minimum version of Redis used when databases are created or upgraded.  Typically changed in conjunction with the `redis_upgrade_policy` setting.<br/><br/>  The value parameter should be a version number in form of "x.y" where _x_ represents the major version number and _y_ represents the minor version number.  The final value corresponds to the desired version of Redis. |
```
Comment (rrelledge) [inline content/rs/references/rladmin.md:712]:
```suggestion
| default_redis_version <value> | The minimum version of Redis used when databases are created or upgraded.  Typically changed in conjunction with the `redis_upgrade_policy` setting.<br/><br/>  The value parameter should be a version number in the form of "x.y" where _x_ represents the major version number and _y_ represents the minor version number.  The final value corresponds to the desired version of Redis. |
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I just left one small suggestion for a possible missing word.

**Review by kaitlynmichael — APPROVED** [review]

File: content/rs/security/internode-encryption.md (hunk 1 above)
Comment (lanceleonard) [reply content/rs/security/internode-encryption.md]: Fixed in latest commit.

File: content/rs/references/rladmin.md (hunk 2 above)
Comment (lanceleonard) [reply content/rs/references/rladmin.md]: Fixed in latest commit.

**Pull request merged**